test/js-native-api/test_reference/test_reference.c

```diff
@@ -1,4 +1,4 @@
-#define NAPI_EXPERIMENTAL
+#define NAPI_VERSION 9
```

Comment:
If the reason for `#define NAPI_EXPERIMENTAL` is that we want to test an API that is defined conditionally on it, we should just  remove the directive, like we did for version 8: https://github.com/nodejs/node/commit/d15475578abfcc7e0a420719b8d55a6b3ab4f198#diff-1a6808fd8fc22b93d265cab3a30ebef378786765e9d76fa2b05c9a3188615fa6L1

Reply:
The default `NAPI_VERSION` no longer bumps when a new Node-API version is released: https://github.com/nodejs/node/blob/main/src/js_native_api.h#L20. So starting from Node-API 9, we'll have to explicitly define the `NAPI_VERSION` in tests.

Reply:
I tried locally to remove the NAPI_EXPERIMENTAL and it worked, so the NAPI_VERSION seen by the addon must be the released version, not the baseline version. This makes sense, because addons have to store the released version when they get compiled, not the baseline version.

Reply:
Oh, I see now that it had a bunch of implicit declaration warnings. NM.